types/testingutils/comparable/helpers.go

```diff
@@ -31,3 +35,21 @@ func FixIssue178(input *types.ConsensusData, version spec2.DataVersion) *types.C
 
 	return ret
 }
+
+// UnmarshalSSVStateComparison reads a json derived from 'test' and unmarshals it into 'targetState'
+func UnmarshalSSVStateComparison(testName string, folderName string, targetState types.Root) (types.Root, error) {
```

Comment:
why is this used by just one function? don't we have a bunch of those calls?

Reply:
While adding missing `getPostState()` I was laying down infra for farther refactoring.
Didn't want to do it all now to not distract from #274

Reply:
Opened #283 to track refactoring changed.
I want to solve each issue in ints own PR to cause less disruptions and ensure quality